Resources/Prototypes/_Goobstation/Wizard/spellbook_catalog.yml

```
@@ -795,6 +795,16 @@
   - SpellbookEquipment
   disableRefund: true
 
+- type: listing
+  id: Spellbook-fire-guitar
+  name: Spellbook-fire-guitar-name
+  description: Spellbook-fire-guitar-desc
+  productEntity: FierGuitarInstrument
```

```suggestion
  productEntity: FireGuitarInstrument
```
+ Перекинь в нашу папку.